dockerfiles/run/guice/cassandra-ldap/destination/conf/batchsizes.properties

```diff
@@ -0,0 +1,4 @@
+metadata=200
```

Comment:
I only find it sad that we loose information about the fact that it's for FETCH.

Thus I will use these properties: 

```
fetch.headers=200
fetch.body=100
fetch.full=50
fetch.metedata=200
```